lib/helpers.js

```diff
@@ -45,4 +50,30 @@ module.exports = {
             }
         });
     },
+
+    getSystemAttribute() {
+        const osType = os.type();
+        const osArchitecture = os.arch();
+        const RAMSize = os.totalmem();
+        const browser = DetectBrowser.detect();
+        const systemAttr = [{
+            key: 'client',
+            value: `${PJSON_NAME}|${PJSON_VERSION}`,
+            system: true,
+        }, {
+            key: 'os',
+            value: `${osType}|${osArchitecture}`,
+            system: true,
+        }, {
+            key: 'RAMSize',
+            value: `${RAMSize}`,
+            system: true,
+        }, {
+            key: 'browser',
```

Comment:
I suppose that information about browser should be provided by test runner. Node version could be usefull information, but should be setted to another attribute.